### ISSUE TITLE
SelectOneMenu: Advanced Mode ARIA Issue

### ISSUE THREAD

**Issue by melloware** [primefaces/primefaces#8473]
**Describe the defect**
When using Advanced Mode in showcase the following ARIA issue is reported:

>  Attribute aria-activedescendant value should either refer to a descendant element, or should be accompanied by attribute aria-owns.

![image](https://user-images.githubusercontent.com/4399574/154959358-aef9b11f-3fca-4326-832a-43f42313c51e.png)

See: https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Roles/combobox_role
